Cargo.toml

```diff
@@ -2,7 +2,7 @@
 name = "drop"
 version = "0.1.0"
 authors = [ "Ogier Bouvier <ogier@bouvier.family", "Matteo Monti <matteo.monti@epfl.ch>" ]
-edition = "2018"
+edition = "2021"
```

Comment:
New rust! Cool!

Reply:
Yeah, [not many new features](https://doc.rust-lang.org/edition-guide/rust-2021/index.html) but a few quirks were fixed. And it stays compatible with crates built in previous editions so we can slowly update the crates :)